src/coreclr/src/vm/gchelpers.cpp

```diff
@@ -536,9 +440,20 @@ OBJECTREF AllocateSzArray(MethodTable* pArrayMT, INT32 cElements, GC_ALLOC_FLAGS
             orDummyObject->SetMethodTable(g_pObjectClass);
         }
         else
-#endif  // FEATURE_64BIT_ALIGNMENT
-
         {
+#else   // ! FEATURE_64BIT_ALIGNMENT
```

Comment:
Two ifdefs may be more readable than #if / #else / #endif. #ifdefs around unmatched `{` are hard to read.

Reply:
It was also causing build issues on arm.